projects/disagreement/run.py

```diff
@@ -160,7 +158,19 @@ def make_env_all_params(rank, args):
     device = torch.device(dev_name)
     print("device: " + str(device))
 
+    # Verify if checkpoint dir is given and define dir where model will be savedd
+    if "CHECKPOINT_DIR" not in os.environ:
+        raise KeyError("Environment variable CHECKPOINT_DIR not found, required.")
+    else:
+        checkpoint_dir = os.path.join(os.environ["CHECKPOINT_DIR"], "robot_arm")
+
+    model_dir = os.path.join(
+        checkpoint_dir, f"{run_args.exp_name}_{logging_args.project_id}")
```

Comment:
To continue wandb logging after loading we should also add the project_id to have it consistent and not overwrite those runs either.